README.md

```
@@ -182,10 +182,48 @@ function App() {
+import Button from './components/Button';
+
+<Button
+  label="编程语言"
+  category="language"
+  isActive={true}
+  onClick={() => handleFilter('language')}
+/>
+```
+
+**Props:**
+| Prop | Type | Default | Description |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)